ros2cli/ros2cli/daemon/__init__.py

```diff
@@ -69,6 +69,14 @@ def main(*, script_name='_ros2_daemon', argv=None):
                 _print_invoked_function_name(node.get_topic_names_and_types))
             server.register_function(
                 _print_invoked_function_name(node.get_service_names_and_types))
+            server.register_function(
+                _print_invoked_function_name(node.get_publisher_names_and_types_by_node))
+            server.register_function(
+                _print_invoked_function_name(node.get_subscriber_names_and_types_by_node))
+            server.register_function(
+                _print_invoked_function_name(node.get_service_names_and_types_by_node))
+            server.register_function(
+                _print_invoked_function_name(node.get_client_names_and_types_by_node))
```

Comment:
`count_publishers` and `count_subscribers` (which are also graph related), should be added too.

Reply:
True, and I'll need it for `ros2 topic info`. Added in https://github.com/ros2/ros2cli/pull/420/commits/5a1c3136ecf8099f889ad9fc4c9edb317c94edf1.